requirements.txt

```diff
@@ -59,3 +59,5 @@ retrying==1.3.3
 
 httplib2~=0.9.2
 unidecode
+
+jsonpath_rw==1.4.0
```

Comment:
how about move the dependency in extra(https://github.com/lyft/amundsendatabuilder/blob/master/setup.py#L12) as not every databuilder needs this deps?

This will make the project deps easy to manage.

Reply:
Great idea. Will do that :)

Reply:
Done